docs/system/sdk.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Software Development Kits (SDKs)
+tags:
+  - HMX
+  - MX-4
+  - CT
+  - MX-V
+  - T30
+  - T30 FR
+  - C61
+  - Development
+---
+
+## Overview
+
+Reference SDKs are available at 
+[hostmobility.org/sdk](https://hostmobility.org/sdk). 
+
+Legacy toolchains are available at [hostmobility.org:8008/tools](http://hostmobility.org:8008/tools/)
+
+## Building
+
+SDKs are built against the image it will be used with using `bitbake -c populate_sdk <image-name>`. See [sdk](sdk.md).
+
+## Installation
+
+The SDK is installed by running the installer from the build step above. An installation folder is requested and you can use any directory that you have write access to, e.g. `~/sdk` in your home directory.
```

Comment:
Is it installed by running `bitbake -c populate_sdk <image-name>` or is an additional command required? I can't see a specific installer being mentioned.

Reply:
old, has console image as the example.